Cargo.toml

```diff
@@ -11,22 +11,27 @@ edition = "2021"
 rust-version = "1.67.0"
 
 [features]
-default = []
-cli = ["dep:clap"]
+default = ["std"]
+std = []
+cli = ["std", "dep:clap"]
 
 [[bin]]
 name = "stellar-strkey"
 path = "src/bin/stellar-strkey/main.rs"
 required-features = ["cli"]
 doctest = false
 
-[build_dependencies]
+[build-dependencies]
 crate-git-revision = "0.0.6"
 
 [dev-dependencies]
 proptest = "1.0.0"
 
 [dependencies]
-base32 = "0.4.0"
-thiserror = "1.0.36"
```

Comment:
The `thiserror` crate is rarely used in this project, and it does not support `no_std`, so I choose to remove it.

Reply:
I think this is the right call. 👍🏻